test/standard_unit/arrhenius_rate_vector_test_base.h

```diff
@@ -0,0 +1,123 @@
+//-----------------------------------------------------------------------bl-
+//--------------------------------------------------------------------------
+//
+// Antioch - A Gas Dynamics Thermochemistry Library
+//
+// Copyright (C) 2014 Paul T. Bauman, Benjamin S. Kirk, Sylvain Plessis,
+//                    Roy H. Stonger
+// Copyright (C) 2013 The PECOS Development Team
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the Version 2.1 GNU Lesser General
+// Public License as published by the Free Software Foundation.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc. 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301  USA
+//
+//-----------------------------------------------------------------------el-
+
+#ifndef ANTIOCH_ARRHENIUS_RATE_VECTOR_TEST_BASE_H
+#define ANTIOCH_ARRHENIUS_RATE_VECTOR_TEST_BASE_H
+
+#include "antioch_config.h"
+
+#ifdef ANTIOCH_HAVE_CPPUNIT
+
+// C++
+#include <limits>
+
+// Antioch
+#include "antioch/physical_constants.h"
+#include "antioch/units.h"
+
+// Base class
+#include "arrhenius_rate_test_helper.h"
+#include "reaction_rate_vector_test_base.h"
+
+template<typename PairScalars>
+class ArrheniusRateVectorTestBase : public ArrheniusRateTestHelper</*Scalar*/typename Antioch::value_type<PairScalars>::type>,
```

Comment:
It's not clear that `ArreniusRateTestHelper` is actually needed.  Is there a reason against putting all the members of `ArrheniusRateTestHelper` in `ArrheniusRateVectorTestBase`?